app/services/BackgroundTaskService.ts

```diff
@@ -1,41 +0,0 @@
-import BackgroundFetch from 'react-native-background-fetch';
-
-import { INTERSECT_INTERVAL } from '../constants/history';
-import IntersectService from './IntersectService';
-
-class BackgroundTaskService {
-  start() {
-    // Configure it.
-    console.log('creating background task object');
-    BackgroundFetch.configure(
-      {
-        minimumFetchInterval: INTERSECT_INTERVAL,
-        // Android options
-        forceAlarmManager: false, // <-- Set true to bypass JobScheduler.
-        stopOnTerminate: false,
-        startOnBoot: true,
-        requiredNetworkType: BackgroundFetch.NETWORK_TYPE_NONE, // Default
-        requiresCharging: false, // Default
-        requiresDeviceIdle: false, // Default
-        requiresBatteryNotLow: false, // Default
-        requiresStorageNotLow: false, // Default
-      },
-      async (taskId) => {
-        console.log('[js] Received background-fetch event: ', taskId);
-        IntersectService.checkIntersect(null, false);
```

Comment:
After comments around adding the `NativeModules.AsyncStoreManager.trim()` to this, I think we're keeping this separate still